course-mw/sunbird-util/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/oci/OciConnectionManager.java

```diff
@@ -0,0 +1,57 @@
+package org.sunbird.common.models.util.oci;
```

Comment:
Why this class is required here.

Reply:
originally this was added as there were csp specific implementation as part of tag release-5.0.0_RC2.  I think in the later revisions this was removed ( tag release-5.0.1_RC3)
i moved this oci folder from this pr